src/lib/stores/workstreams/types.ts

```diff
@@ -18,27 +20,35 @@ export enum Currency {
 export interface Workstream {
   id: string;
   state: WorkstreamState;
+  applicationsToReview: string[];
+  rejectedApplications?: string[];
+  acceptedApplication?: string;
   creator: string;
+  durationDays: number;
   created_at: Timestamp;
-  payment: Payment;
+  ratePerSecond: Money;
+  total: Money;
   title: string;
   desc: string;
   applicants?: string[];
-  rejectedApplications?: string[];
-  applicationsToReview?: string[];
-  acceptedApplication?: string;
-  duration: number;
 }
 
-export interface Payment {
+export interface Money {
```

Comment:
I somehow feel like this is confusing. Why did you choose this name?

Reply:
Ya Payment didn't really fit. At least at my previous job this currency-amount pair was usually referred to as a "Money type" though that might not be universal :) I think "Rate" is too specific, as this type would also be used later for one-time payments or any monetary value, really. Maybe "Amount" would work but tbh I think just "Money" is descriptive and clear. "MonetaryAmount" seems a bit too verbose :D

Reply:
Ok yeah. I guess I'll get used to it. It just sounds super general to me. Hard to figure out what it is.